TriggerRedHatBuild.groovy

```diff
@@ -0,0 +1,101 @@
+/*
+ * Copyright (c) 2020-present Sonatype, Inc. All rights reserved.
+ * Includes the third-party code listed at http://links.sonatype.com/products/clm/attributions.
+ * "Sonatype" is a trademark of Sonatype, Inc.
+ */
+@Grab('io.github.http-builder-ng:http-builder-ng-core:1.0.4')
+
+import groovyx.net.http.HttpBuilder
+import groovyx.net.http.HttpException
+
+if (args.size() < 3) {
+  println 'Usage: groovy TriggerRedhatBuild.groovy <version> <projectId> <apiKey>'
+  return
+}
+
+def (version, projectId, apiKey) = args
+
+final nextTag = getNextTag(apiKey, projectId, version)
+println "Deploying as ${nextTag}"
+
+build(apiKey, projectId, nextTag)
+
+final completedBuild = getCompletedBuild(apiKey, projectId, nextTag)
+
+println publish(apiKey, projectId, completedBuild.digest, completedBuild.name)
+
+String getNextTag(String apiKey, String projectId, String version) {
+  final tags = HttpBuilder.configure {
```

Comment:
why this way instead of instantiating a client? would save some boilerplate for sure
Examples abound, including https://github.com/sonatype/nexus-internal/blob/c2261683ac73f83efc62bf321c9143c9f9662525/private/testsuite/nexus-testsuite-data/src/main/resources/groovyScripts/rawPopulator.groovy

Reply:
as you noted in slack, this is a newer library, so api is slightly different.